lib/aida/bot.ex

```diff
@@ -86,6 +89,44 @@ defmodule Aida.Bot do
     end
   end
 
+  defp policy_enforcement_message("block", %{"reason" => reason }) do
+    "Policy Enforcement Notification - Bot has been blocked.\n\n#{reason}"
+  end
+
+  defp policy_enforcement_message(action, data) do
+    "Policy Enforcement Notification - Unknown action: #{action}.\n\n#{Poison.encode!(data)}"
+  end
+
+  defp post_notification(notifications_url, notification_type, data) do
```

Comment:
Maybe we can extract all this to a notification handler of some sort. Human override functionality should provide us with another use case for this interaction and it may tell us in which direction we should refactor. Let's just keep an eye open for that.